zuul.d/edpm.yaml

```diff
@@ -11,6 +11,7 @@
 # Virtual Baremetal job with CRC and single compute node.
 - job:
     name: cifmw-crc-podified-edpm-baremetal
+    nodeset: centos-9-stream-crc-4xlarge
```

Comment:
what about downstream?

Reply:
Sry, didn't get what about downstream?

You mean need to create same nodeset in downstream? In downstream job is stable with smaller flavor.

Reply:
~> DS has an override already, so it won't crash with this change